src/main.css

```diff
@@ -0,0 +1,22 @@
+/* Styling for Find A Doc Page  */
+
+label {
+    font-size: 20px;
+}
+label[for=lname] {
+    margin-left: 40px;
+    margin-right: 10px;
+}
+label[for=postal] {
+    margin-left: 38px;
+}
+#city {
+    margin-left: 60px;
+}
+#states {
+    margin-left: 48px;
+}
```

Comment:
You may want to use a the [CSS Grid Layout](https://www.w3schools.com/css/css_grid.asp) to line everything up instead of individual margins. This could also allow you to remove the br tags.

Reply:
Good looks Etienne! Will do.

Reply:
@dazedcodes Can click `Resolve conversation` if the fix is pushed!